[2023-06-08 08:52:45,192] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: 2_daily_transformation_analysis.daily_transform 2020-08-01T00:00:00+00:00 [queued]>
[2023-06-08 08:52:45,204] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: 2_daily_transformation_analysis.daily_transform 2020-08-01T00:00:00+00:00 [queued]>
[2023-06-08 08:52:45,204] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-06-08 08:52:45,205] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2023-06-08 08:52:45,206] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-06-08 08:52:45,220] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): daily_transform> on 2020-08-01T00:00:00+00:00
[2023-06-08 08:52:45,225] {standard_task_runner.py:52} INFO - Started process 1155 to run task
[2023-06-08 08:52:45,229] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '2_daily_transformation_analysis', 'daily_transform', '2020-08-01T00:00:00+00:00', '--job-id', '6', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/transform_and_analysis.py', '--cfg-path', '/tmp/tmpjeva6di1', '--error-file', '/tmp/tmparhua1kd']
[2023-06-08 08:52:45,231] {standard_task_runner.py:77} INFO - Job 6: Subtask daily_transform
[2023-06-08 08:52:45,277] {logging_mixin.py:104} INFO - Running <TaskInstance: 2_daily_transformation_analysis.daily_transform 2020-08-01T00:00:00+00:00 [running]> on host 1e9c7bef4812
[2023-06-08 08:52:45,329] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=2_daily_transformation_analysis
AIRFLOW_CTX_TASK_ID=daily_transform
AIRFLOW_CTX_EXECUTION_DATE=2020-08-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-08-01T00:00:00+00:00
[2023-06-08 08:52:45,330] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2023-06-08 08:52:45,332] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'cd /dbt && dbt run --models transform --profiles-dir .']
[2023-06-08 08:52:45,340] {subprocess.py:74} INFO - Output:
[2023-06-08 08:52:46,856] {subprocess.py:78} INFO - Running with dbt=0.19.0
[2023-06-08 08:52:46,887] {subprocess.py:78} INFO - Encountered an error while reading the project:
[2023-06-08 08:52:46,888] {subprocess.py:78} INFO -   ERROR: Runtime Error
[2023-06-08 08:52:46,889] {subprocess.py:78} INFO -   Could not find profile named 'Airflow_DBT_Snowflake'
[2023-06-08 08:52:46,900] {subprocess.py:78} INFO - Encountered an error:
[2023-06-08 08:52:46,901] {subprocess.py:78} INFO - Runtime Error
[2023-06-08 08:52:46,902] {subprocess.py:78} INFO -   Could not run dbt
[2023-06-08 08:52:47,061] {subprocess.py:82} INFO - Command exited with return code 2
[2023-06-08 08:52:47,074] {taskinstance.py:1501} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1157, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1331, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1361, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/bash.py", line 176, in execute
    raise AirflowException('Bash command failed. The command returned a non-zero exit code.')
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code.
[2023-06-08 08:52:47,077] {taskinstance.py:1551} INFO - Marking task as FAILED. dag_id=2_daily_transformation_analysis, task_id=daily_transform, execution_date=20200801T000000, start_date=20230608T085245, end_date=20230608T085247
[2023-06-08 08:52:47,134] {local_task_job.py:149} INFO - Task exited with return code 1
